aspnet/DateRangePicker/display-format.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: display format
-description: display format
+title: ASP.NET DateRangePicker Display Format | Syncfusion
+description: Learn here all about Display Format use in Syncfusion aspnet DateRangePicker control, its elements, and more.
 platform: aspnet
 control: DateRangePicker
 documentation: ug
 keywords: format
 ---
 
-## Display Format
+# Display format in ASP.NET DateRangePicker Component
```

Comment:
Display format in ASP.NET Web Forms DateRangePicker Component

Reply:
resolved